include/NeoFOAM/finiteVolume/cellCentred/boundary/volume/fixedGradient.hpp

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2023 NeoFOAM authors
+
+#pragma once
+
+#include "Kokkos_Core.hpp"
+
+#include "NeoFOAM/finiteVolume/cellCentred/boundary/volumeBoundaryFactory.hpp"
+#include "NeoFOAM/mesh/unstructured.hpp"
+#include "NeoFOAM/core/parallelAlgorithms.hpp"
+
+namespace NeoFOAM::finiteVolume::cellCentred::volumeBoundary
+{
+
+namespace impl
```

Comment:
i think at a different place we already use namespace `detail` to signal we don't consider something part of the "public" api.

```suggestion
namespace detail
```

Reply:
done